packages/components/src/public/QueryModel/TabbedGridPanel.tsx

```diff
@@ -147,9 +200,24 @@ export const TabbedGridPanel: FC<TabbedGridPanelProps & InjectedQueryModels> = m
                         })}
                     </ul>
                 )}
-
-                <GridPanel key={activeId} actions={actions} asPanel={false} model={activeModel} {...rest} />
+                <GridPanel
+                    key={activeId}
+                    actions={actions}
+                    asPanel={false}
+                    model={activeModel}
+                    onExport={exportHandlers}
+                    advancedExportOptions={advancedExportOptions}
+                    {...rest}
+                />
             </div>
+            {queryModels !=null && showExportModal &&
```

Comment:
nit: Try to avoid using `==` and `!=`. I would recommend `{showExportModal && !!queryModels ...`.